datasets/NRS-Monthly-Data-on-Births-and-Deaths-Registered-in-Scotland/info.json

```diff
@@ -18,6 +18,29 @@
         "stage": [
             "To Do"
         ],
+        "columns": {
+            "Area": {
+                "parent": "http://purl.org/linked-data/sdmx/2009/dimension#refArea",
+                "value": "http://statistics.data.gov.uk/id/statistical-geography/{area}"
+            },
+            "Period": {
+                "parent": "http://purl.org/linked-data/sdmx/2009/dimension#refPeriod",
+                "description": "The information presented is based on the date of registration, not the date on which the event occurred. This is consistent with all routine data published by the National Records of Scotland (NRS) for examplein the Registrar General’s Annual Reports (http://www.nrscotland.gov.uk/statistics-and-data/statistics/stats-at-a-glance/registrar-generals-annual-review). Further information on date of registration versus date of occurrence is given on the Monthly data on Births and Deaths (http://www.nrscotland.gov.uk/statistics-and-data/statistics/statistics-by-theme/vital-events/general-publications/weekly-and-monthly-data-on-births-and-deaths/monthly-data-on-deaths-registered-in-scotland) page of the NRS website.",
+                "value": "http://reference.data.gov.uk/id/month/{year}-{month}"
+            },
+            "Cause of Death": {
+                "label": "Cause of Death",
+                "value": "http://gss-data.org.uk/data/gss_data/covid-19/nrs-monthly-data-on-births-and-deaths-registered-in-scotland#concept/cause-of-death/{+cause_of_death}"
+            },
+            "Value": {
+                "unit": "http://gss-data.org.uk/def/concept/measurement-units/{+measure_type}",
```

Comment:
Again, we might need to split this in to two datasets if anyone gets back to me and confirms that the pipeline cannot host multi-unit output of this form. I can't see anything convincing me it can't cope with it. It isn't quite the same as a multi-measure dataset.